src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesSettings/ExternalDataSourcesSettings.js

```diff
@@ -0,0 +1,372 @@
+            key={`${externalDataSource?.name}-action-delete`}
+            buttonStyle="dropdownItem"
+            data-test-external-data-source-delete
+            marginBottom0
+            onClick={() => {
+              setDeleteModal(true);
+              onToggle();
+            }}
+          >
+            <Icon icon="trash">
+              <FormattedMessage id="stripes-core.button.delete" />
+            </Icon>
+          </Button>
+        </>
+      );
+    }
+
+    return (actionsArray?.length ? actionsArray : null);
+  };
+
+  const renderSettingsHeader = renderProps => (
+    <PaneHeader
+      {...renderProps}
+      lastMenu={perm ?
+        <Button
+          buttonStyle="primary"
+          data-test-external-data-source-new
+          id="clickable-new-external-datasource"
+          marginBottom0
+          onClick={() => setMode(CREATING)}
+        >
+          <FormattedMessage id="ui-local-kb-admin.job.new" />
+        </Button> : null
+      }
+      paneSub={<FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.sourceCount" values={{ count }} />}
+      paneTitle={<FormattedMessage id="ui-local-kb-admin.section.externalDataSources" />}
+    />
+  );
+
+  const renderViewHeader = renderProps => (
+    <PaneHeader
+      {...renderProps}
+      actionMenu={getActionMenu}
+      dismissible
+      onClose={() => setExternalDataSource()}
+      paneTitle={externalDataSource?.name}
+    />
+  );
+
+  return (
+    <>
+      <Pane
+        data-test-external-data-sources
+        defaultWidth="fill"
+        id="settings-external-data-sources"
+        renderHeader={renderSettingsHeader}
+      >
+        <MultiColumnList
+          columnMapping={{
+            name: <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.name" />,
+            type: <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.type" />,
+            uri: <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.uri" />,
+            active: <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.isActive" />,
+            trustedSourceTI: <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.trustedSourceTI" />,
+            syncStatus: <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.syncStatus" />,
+            cursor: <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.cursor" />,
+            lastCheck: <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.lastChecked" />,
+          }}
+          contentData={externalKbs}
+          formatter={{
+            active: data => { return <FormattedMessage id={data.active ? 'ui-local-kb-admin.yes' : 'ui-local-kb-admin.no'} />; },
+            trustedSourceTI: data => { return <FormattedMessage id={data.trustedSourceTI ? 'ui-local-kb-admin.yes' : 'ui-local-kb-admin.no'} />; },
+          }}
+          onRowClick={(_e, eds) => setExternalDataSource(eds)}
```

Comment:
We should follow the new pattern, this shouldn't be row click, but should instead be a Button of type "link" on the name column. Ask @CalamityC if you need an example, as she just implemented this for pick lists in kint-components